Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
+    jwt_signed_request (2.5.1)
+      jwt (>= 1.5.0, < 2.2.0)
       rack
     loofah (2.2.3)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.1)
-    mini_portile2 (2.3.0)
+    method_source (0.9.2)
```

Comment:
### [method_source](http://banisterfiend.wordpress.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.9.1 → 0.9.2

[_~~change-log~~, [source-code](https://github.com/banister/method_source)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/banister/method_source/compare/v0.9.1...v0.9.2).

These are the individual commits:
- (48405c4) [Revert "method_source: fix broken Procs on JRuby 9.2.0.0"](https://github.com/banister/method_source/commit/48405c48cd6c4054bdec187530a6d66b93a2b57b)
- (1ff2066) [Merge pull request #54 from banister/52-jruby-patch-removal](https://github.com/banister/method_source/commit/1ff206639c92b1c3c734d29f2e8257cd0a88a795)
- (6259c06) [Release v0.9.2](https://github.com/banister/method_source/commit/6259c06d87ee00ef8ab5a3897e41e6cd64a06378)
- (ba41c6b) [Merge pull request #55 from banister/release-0-9-2](https://github.com/banister/method_source/commit/ba41c6b555c7ba75dc7816e990d6cd85a8152132)


</details>